auto_submit/lib/service/config.dart

```diff
@@ -70,9 +70,14 @@ class Config {
       githubInstallationUri,
       headers: headers,
     );
-    final list = json.decode(response.body).map((data) => (data) as Map<String, dynamic>).toList();
-    final String installatinId = list[0]['id'].toString();
-    return installatinId;
+    final List<dynamic> list = json.decode(response.body).map((data) => (data) as Map<String, dynamic>).toList();
+    late String installationId;
+    for (Map<String, dynamic> installData in list) {
+      if (installData['account']!['login']!.toString() == 'flutter') {
```

Comment:
Instead of hard coding 'flutter', can we pass the `RepositorySlug` in `getInstallationId`?

That way this works for teams outside of flutter

Reply:
Sure! I added the repository slug.